src/openapi.ts

```diff
@@ -4,7 +4,7 @@ import {
   SemanticReleasePlugin,
 } from "~/_config";
 
-const openApiSpecGlob = "spec/**/openapi.yaml";
+const openApiSpecGlob = "spec/*.yaml";
```

Comment:
can we still have it be something like `spec/**/*.yaml`?  Our payments API is current down another directory under spec so this will break us, I'm pretty sure.